test/TypeSynonyms.agda

```diff
@@ -1,5 +1,6 @@
+open import Haskell.Prim using (Type)
 
-data Nat : Set where
+data Nat : Type where
   Zero : Nat
   Suc  : Nat → Nat
 {-# COMPILE AGDA2HS Nat #-}
```

Comment:
You missed a couple of occurrences of `Set` below.

Reply:
Oops. Should be fixed now.